airflow/providers/google/cloud/hooks/gcs.py

```diff
@@ -272,7 +272,7 @@ def download(
         chunk_size: Optional[int] = None,
         timeout: Optional[int] = DEFAULT_TIMEOUT,
         num_max_attempts: Optional[int] = 1,
-    ) -> Union[str, bytes]:
+    ) -> bytes:
```

Comment:
I think this is still `Union[str, bytes]` because `filename` should be a str.

Reply:
You are right. Fixed